docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
+2. Open starter code in Colab Notebook (badge-link below) and run it as a "`neptuner`" user - zero setup, it just works.
+3. View starter code as Python script on |script|.
 
-    class CoolSystem(pl.LightningModule):
+|Run on Colab|
 
-        def __init__(self):
-            ...
+Before you start
+^^^^^^^^^^^^^^^^
 
-        def forward(self, x):
-            ...
+**Prerequisites**
 
-        def training_step(self, batch, batch_idx):
-            ...
+* Python 3,
+* Neptune account |register|,
+* Minimal familiarity with the PyTorch Lightning.
 
-        def configure_optimizers(self):
-            ...
+**Supported version**
 
-        @pl.data_loader
-        def train_dataloader(self):
-            ...
+* ``pytorch-lightning>=0.9.0``
+* ``neptune-client>=0.4.105``
 
-        @pl.data_loader
-        def val_dataloader(self):
-            ...
+Installation
+^^^^^^^^^^^^
+**Install neptune-client**
 
-Create the **NeptuneLogger** with all the information you want to track
------------------------------------------------------------------------
-.. code-block:: python3
+From PyPI:
 
-    from pytorch_lightning.loggers.neptune import NeptuneLogger
+.. code-block:: bash
 
-    neptune_logger = NeptuneLogger(
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        experiment_name="default",  # Optional,
-        params={"max_epochs": 10,
-                "batch_size": 32},  # Optional,
-        tags=["pytorch-lightning", "mlp"]  # Optional,
-    )
-
-Create the **Trainer** and pass **neptune_logger** to logger
-------------------------------------------------------------
-.. code-block:: python3
+    pip install neptune-client -U
 
-    from pytorch_lightning import Trainer
+From Conda:
 
-    model = CoolSystem()
-    trainer = Trainer(max_epochs=10, logger=neptune_logger)
+.. code-block:: bash
 
-    trainer.fit(model)
+    conda install neptune-client -c conda-forge
 
-Log additional information after the **.fit** loop ends
--------------------------------------------------------
+**Install PyTorch Lightning and torchvision**
 
-You can log additional metrics, images, model binaries or other things after the ``.fit`` loop is over.
-You just need to specify ``close_after_fit=False`` in the ``NeptuneLogger`` initialization.
+From PyPI:
 
-.. code-block:: python3
+.. code-block:: bash
 
-    neptune_logger = NeptuneLogger(
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        close_after_fit=False,
-        ...
-    )
+    pip install pytorch-lightning torchvision
 
-**Log test metrics**
+From Conda:
 
-.. code-block:: python3
+.. code-block:: bash
+
+    conda install pytorch-lightning -c conda-forge
+    conda install pytorch torchvision -c pytorch
 
-    trainer.test(model)
+Now, you have all dependencies installed. Let's move to the actual integration.
 
-**Log additional metrics**
+Step 1 - Necessary Imports
```

Comment:
I think it should be an action so 

Step 1 - import PyTorch libs and lightning

Also I think we are using Step 1: in other places (could you check quickstarts? )

Reply:
I will follow `Step 1:` - thanks